rpcs3/Emu/Cell/Modules/cellSpursSpu.cpp

```diff
@@ -133,7 +133,7 @@ bool spursDma(spu_thread& spu, u32 cmd, u64 ea, u32 lsa, u32 size, u32 tag)
 
 	if (cmd == MFC_GETLLAR_CMD || cmd == MFC_PUTLLC_CMD || cmd == MFC_PUTLLUC_CMD)
 	{
-		const u32 rv = spu.get_ch_value(MFC_RdAtomicStat);
+		const u32 rv = ::narrow<u32>(spu.get_ch_value(MFC_RdAtomicStat));
```

Comment:
This is very wrong and would trigger an assertation in some cases, use static_cast.